sihl-email/src/sihl_email.ml

```diff
@@ -160,31 +173,55 @@ module Smtp : Sihl.Contract.Email.Sig = struct
 
   let send email = intercept send' email
   let bulk_send _ = failwith "Bulk sending not implemented yet"
-
-  let start () =
-    (* Make sure that configuration is valid *)
-    if Sihl.Configuration.is_production ()
-    then Sihl.Configuration.require schema
```

Comment:
@jerben I removed this to make sure configs can be passed in without requiring env vars.